src/ert/cli/main.py

```diff
@@ -43,16 +44,25 @@ def run_cli(args: Namespace, plugin_manager: Optional[ErtPluginManager] = None)
     # Changing current working directory means we need to update
     # the config file to be the base name of the original config
     args.config = os.path.basename(args.config)
+    forward_models = []
 
     ert_config = ErtConfig.with_plugins().from_file(args.config)
 
     local_storage_set_ert_config(ert_config)
+    counter_fm_steps = Counter(fms.name for fms in ert_config.forward_model_steps)
 
     # Create logger inside function to make sure all handlers have been added to
     # the root-logger.
     logger = logging.getLogger(__name__)
     for fm_step in ert_config.forward_model_steps:
```

Comment:
You can simplify this loop to `for fm_step_name, count in conter_fm_steps.items():` now.